screen/wallets/CreateRecoveryPhraseScreen.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { View, Text, StyleSheet, TouchableOpacity } from 'react-native';
+import { useExtendedNavigation } from '../../hooks/useExtendedNavigation';
+import { RouteProp, useRoute } from '@react-navigation/native';
+import { DetailViewStackParamList } from "../../navigation/DetailViewStackParamList";
+import { NativeStackNavigationProp } from '@react-navigation/native-stack';
+import { CommonActions } from '@react-navigation/native';
+import { OnboardingStackParamList } from '../../navigation/OnboardingStack';
+
+type NavigationProps = NativeStackNavigationProp<DetailViewStackParamList, 'WalletsList'>;
+
+const CreateRecoveryPhraseScreen = () => {
```

Comment:
This should be renamed to `CreateWalletScreen` and the create wallet logic should be moved here.